testsuite/features/step_definitions/common_steps.rb

```diff
@@ -418,7 +426,8 @@
   node = get_target(host)
   arch, _code = node.run('uname -m')
   arch.chomp!
-  cmd = "zgrep '#{text}' /var/cache/zypp/raw/susemanager:test-channel-#{arch}/repodata/*updateinfo.xml.gz"
+  # TODO: adapt for architectures
+  cmd = "zgrep '#{text}' /var/cache/zypp/raw/susemanager:test-channel-for-sle/repodata/*updateinfo.xml.gz"
```

Comment:
Yeah, I think this TODO must be done before we merge this PR.
This step might probably be used also in BV.

Reply:
It is apparently currently not used anymore as it was only for the client, I'll still adapt it before merging so it will be ready

Reply:
On master branch the corresponding line  seems to not make any sense since it refers to `zypp` directory with various architectures https://github.com/uyuni-project/uyuni/blob/master/testsuite/features/step_definitions/common_steps.rb#L421

I would rather see this fixed in a different PR, I can drop changes related to it in this PR, mostly since it is only relevant to 4.3 and 4.2 now